### DISCUSSION TITLE
[Request] Show new models uploads of followed creators

### DISCUSSION THREAD

**Discussion by CesarCruzUtec** [BlafKing/sd-civitai-browser-plus#147] [Ideas]
Would it be possible to add an option to load in the mod browser, the new models from creators that I follow on civitai. The page already does it with the notifications, so probably can be implemented with the API key?
![image](https://github.com/BlafKing/sd-civitai-browser-plus/assets/67798431/23ec101d-068b-45d2-9f2d-63bcc2bee49d)

**Comment by BlafKing** [comment]
Thanks for writing in!
This functionality unfortunately cannot be implemented by me since it relies on the functionality of [CivitAI's REST API](https://github.com/civitai/civitai/wiki/REST-API-Reference), the CivitAI website uses a different version than the public REST API, so that's why the functions available on the website differ from the public API.

As far as I can tell this is currently not possible on the public API, so it would need to be implemented there first before I can add it to the extesnion.

If you would like, you can post a [feature request on the CivitAI Github](https://github.com/orgs/civitai/discussions/new?category=ideas) to request this feature for the public API! :)
